Pipfile

```diff
@@ -17,8 +17,8 @@ quandl = "==3.4.8"
 statsmodels = "==0.10.1"
 bs4 = "==0.0.1"
 tweepy = "==3.5.0"
+scipy = "*"
```

Comment:
Please can you pin this requirement to the version you are using (with `pip freeze`) and then re lock the pipfile.lock (`pipenv lock`)